videoreviewsAPI.md

```diff
@@ -20,7 +22,29 @@ The integration automatically provides:
 - Above-fold summary button linking to review block
 - Zero-impact when no videos are available
 
-## Integration Steps
+## Integration Steps Expeerly for your Image Gallery/Carousel 
+
+### Step 1: Get API acess
+
+~Log into your Expeerly dashboard and copy the API token.~ PENDING 
+
+For the beta phase, get in touch with the expeerly team directly at product@expeerly.com. The API is publicly available for now.
+
+### Step 2: Install Mux Player (Web, iOS, Android)
+Get the mux player that best fits your needs [here](https://www.mux.com/docs/guides/play-your-videos).
+
+### Step 3: Call the expeerly API and pass your Store-ID
+Call the expeerly API `https://app.expeerly.com/api/1.1/wf/get-product-videos-processed/?gtin=${GTIN/UPCnumber}`
+
```

Comment:
The response is an array of videos. For setting up the mux player you will need the `mux_playback_id_text`
```
[
    {
        ...
        "mux_playback_id_text": "EfufB005xJ3uTDV2a9oMgCHzNuAjAqdvk018j8Wzcu4HA",
    },
]
```

Reply:
It might makes sense to provide a working example on the [expeerly.com/demo.html](expeerly.com/demo.html) page